specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/secrets.json

```diff
@@ -744,6 +743,21 @@
       },
       "description": "A secret consisting of a value, id and its attributes."
     },
+    "SecretBundleWithRotationPolicy": {
+      "allOf": [
+        {
+          "$ref": "#/definitions/SecretBundle"
+        }
+      ],
+      "properties": {        
+        "rotation_policy": {
+          "$ref": "#/definitions/SecretRotationPolicy",
```

Comment:
Why not just return a policy on SecretBundle like you do on KeyBundle? Not only can we not work around issues this might create for the SDK (see previous comments), but this is a departure from the pattern already established with Keys.